lib/src/server/server.dart

```diff
@@ -135,10 +160,14 @@ class Server {
         _connections.remove(connection);
       });
     }, onError: (error) {
-      print('Socket error: $error');
+      _printSocketError(error);
     });
   }
 
+  void _printSocketError(String error) {
```

Comment:
This should be `Object error` (because onError handler would pass non-strings in here). This actually caused some breakage internally when first committed, the fact that it is not causing issue here means our test coverage is not stellar

Reply:
Done.